webrtc.html

```diff
@@ -359,6 +359,16 @@
             the application to provide information about whether it wishes this
             type of processing enabled or disabled.</p>
           </dd>
+
+          <dt>sequence&lt;RTCRtpCodecCapability&gt; codecs</dt>
+          <dd>
+            <p>This establishes what codecs the browser is permitted to
+            negotiate.  Codec capabilities that do not appear in this list
+            cannot be negotiated.</p>
+            <p>The order of this sequence determines the preference order.
+            Codecs that appear first in this list MUST be selected by the user
+            agent in preference to those that appear after.</p>
+          </dd>
```

Comment:
How does this work if this option is not specified?
I think the text should say that "if left off, browser can negotiate everything".